### PR TITLE
Adding CSV support to the documentation

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -376,20 +376,31 @@ mattermost config validate
...
 
 mattermost export
-------------------------
+-----------------
+
+  Description
+    Compliance export commands
+
+  Child Commands
+    -  `mattermost export schedule`_ - Add users to a channel
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: `Add users to a channel` --> `Schedule an export job`

File: source/administration/command-line-tools.rst [hunk 2]
```diff
@@ -376,20 +376,31 @@ mattermost config validate
...
 
   Description
-    Export data from Mattermost in a format suitable for importing into a third-party archive system.
+    Schedule an export job in a format suitable for importing into a third-party archive system.
 
   Format
     .. code-block:: none
 
-      mattermost export
+      mattermost export schedule
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: I assume we'll want to release note this as it could break existing scripts?

**Comment by jasonblais** [comment]
I added three commits 
 - https://github.com/mattermost/docs/pull/2112/commits/ca10a2810ae65c327cef4ed07fdc6b9d2f91f3e6
 - https://github.com/mattermost/docs/pull/2112/commits/c372a502d5dff1dcc06174fdff76760b53920de7
 - https://github.com/mattermost/docs/pull/2112/commits/50b784bec9a2314e0261a94b51749e1009251226

Three questions:
1) Do we want to release note the CLI change (`mattermost export` to `mattermost export schedule` as it could break existing scripts?
2) Do we want to add steps on how to migrate from the old system to the new one?
3) Would it be beneficial including a sample export for each file format, stored in the docs repo? This way a customer can see them before turning the feature on.

**Comment by jasonblais** [comment]
1. Sounds good, @amyblais let's include a note about this change.
2. Right, that's true. Does this list cover the changes

**posts.csv**
1. adds join/leave messages, which can be used to monitor who could have read messages posted in a channel
2. adds team ID, channel ID and user ID to make it easier to distinguish who posted what, and where
3. adds post type and channel type
4. removes some unnecessary fields such as PostProps

**meta.json**
1. adds channel information
2. removes unused fileds such as "keywords"

**files folder** (new)
1. the ZIP file now includes all files uploaded to the system. Right now they are separated into folders by the post ID, but we plan to combine them all in a single folder



3. Makes sense, can you help attach your earlier prototype and include a link to it in the doc?

**Comment by jespino** [comment]
2. Not exactly, this would be the complete list of changes

**posts.csv**
- adds join/leave messages, which can be used to monitor who could have read messages posted in a channel
- adds team ID, channel ID and user ID to make it easier to distinguish who posted what, and where
- adds post type and channel type
- adds attachments as lines in the CSV instead of PostFileIds field
- adds edited posts information instead of the PostUpdateAt, and reference to the Post which overwrite the current post as "Post Edited By" column (The old PostOriginalId).
- removes some unnecessary fields such as UserNickname, PostProps, PostFileIds, PostHashtags, PostUpdateAt, PostDeleteAt, PostRootId, PostParentId and PostOriginalId.

**meta.json**
- adds channel information and attachments count.
- removes unused fileds such as "keywords", "desc", "status", "user_id", "create_at", "emails" and "type"

**files folder** (new)
- the ZIP file now includes all files uploaded to the system. Right now they are separated into folders by the post ID, but we plan to combine them all in a single folder.

3. I can generate a simple example for this, to be sure that we are up-to-date.

**Comment by jasonblais** [comment]
Thanks! 

We're making a few changes to the export file, so I'll wait until those are merged before making updates.

@jespino also, if you can share a simple example next week, I can include in the PR.

**Comment by jasonblais** [comment]
A customer is asking for this documentation, so I'll merge it given we're almost ready for v5.1,

**Pull request merged**